core/tests/widgets/test_imageview.py

```diff
@@ -171,3 +184,16 @@ def test_rehint_empty_image(params):
     assert width == 0
     assert height == 0
     assert aspect_ratio is None
+
+
+def test_as_image_toga():
+    toga_img = toga.Image(ABSOLUTE_FILE_PATH)
+    imageview = toga.ImageView(toga_img)
+    toga_img_2 = imageview.as_image()
+    assert toga_img_2.size == (32, 32)
+
+
+def test_as_image_pil():
+    imageview = toga.ImageView(ABSOLUTE_FILE_PATH)
+    pil_img = imageview.as_image(PIL.Image.Image)
```

Comment:
Might want to add a check that the image actually *is* a PIL image - this test would pass if you returned a Toga Image, because the `.size` attribute is honoured by both.